File: swagger/models/upload_client_photo_request.py
from .base_model import BaseModel


class UploadClientPhotoRequest(BaseModel):
	"""
	no description available

	Attributes:

		bytes
		A Base64-encoded string representation of the image’s byte array.

		client_id
		The RSSID of the client for whom the photo is to be uploaded.
	"""

	input_map = {
		'Bytes': 'bytes',
		'ClientId': 'client_id',
		}

	output_map = {
		'bytes': 'Bytes',
		'client_id': 'ClientId',
		}

	bytes = None
	client_id = None



	def __init__(self, bytes, client_id):
		self.bytes = bytes
		self.client_id = client_id

File: swagger/Models/UploadClientPhotoRequest.php
<?php

namespace App\MindBody\Models;

/**
 * no description available
 */
class UploadClientPhotoRequest extends BaseModel
{
	/**
	 * A Base64-encoded string representation of the image’s byte array.
	 * 
	 * @var string format:byte
	 */
	public $bytes;
	/**
	 * The RSSID of the client for whom the photo is to be uploaded.
	 * 
	 * @var string 
	 */
	public $clientId;
	/**
	 * Constructor
	 */
	public function __construct($bytes, $clientId)
	{
		$this->bytes = $bytes;
		$this->clientId = $clientId;
	}

	protected function getInputMap()
	{
		return [
			'Bytes' => ['bytes', 'string', 'byte'],
			'ClientId' => ['clientId', 'string', null],
			];
	}

	protected function getOutputMap()
	{
		return [
			'bytes' => ['Bytes', 'string', 'byte'],
			'clientId' => ['ClientId', 'string', null],
			];
	}
}
